scripts/test_duplicate_prevention.py

```
@@ -0,0 +1,219 @@
+        unique_messages = set()
+        
+        for result in results1:
+            print(f"  요청 {result['request_id']}: {result['status']}")
+            if result['status'] == 'success':
+                print(f"    세션 ID: {result['session_id']}")
+                print(f"    메시지 ID: {result['message_id']}")
+                print(f"    응답시간: {result['response_time']:.3f}초")
+                unique_sessions.add(result['session_id'])
+                unique_messages.add(result['message_id'])
+        
+        print(f"\n📈 통계 1:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)